classes/UploadUsers.php

```diff
@@ -318,7 +318,9 @@ function uploadUser($record) {
 
 				if ($this->notification) {
 					$subject = elgg_echo('upload_users:email:subject', array(elgg_get_config('sitename')));
-					$message = elgg_echo('upload_users:email:message', array($record['name'], elgg_get_config('sitename'), $record['username'], $record['password'], elgg_get_site_url()));
+					$custom_message = elgg_get_plugin_setting('custom_message', 'upload_users');
+					if (!$custom_message) { $custom_message = ''; }
```

Comment:
Tertiary operations should span across multiple lines.
In this particular case, `elgg_get_plugin_setting()` takes a third argument for `$default` value, so you don't need to use `if`